orttraining/orttraining/test/python/orttraining_test_utils.py

```diff
@@ -86,36 +144,37 @@ def run_test(model, model_desc, device, args, gradient_accumulation_steps, fp16,
                 batch = batch + [learning_rate, ]
             if not use_internal_loss_scaler and fp16:
                 batch = batch + [loss_scale, ]
-            outputs = model(*batch)
+            outputs = model.train_step(*batch)
         elif batch_args_option == BatchArgsOption.Dict:
             args, kwargs = split_batch(batch, model_desc.inputs_, 0)
             if not use_internal_get_lr_this_step:
                 kwargs['Learning_Rate'] = learning_rate
             if not use_internal_loss_scaler and fp16:
                 kwargs[model.loss_scale_input_name] = loss_scale
-            outputs = model(*args, **kwargs)
+            outputs = model.train_step(*args, **kwargs)
         else:
             args_count = int(len(model_desc.inputs_) / 2)   # approx helf args, half kwargs
             args, kwargs = split_batch(batch, model_desc.inputs_, args_count)
             if not use_internal_get_lr_this_step:
                 kwargs['Learning_Rate'] = learning_rate
             if not use_internal_loss_scaler and fp16:
                 kwargs[model.loss_scale_input_name] = loss_scale
-            outputs = model(*args, **kwargs)
+            outputs = model.train_step(*args, **kwargs)
 
         print(outputs[0])
 
     # eval
-    model.eval()
+    if not use_new_api:
+        model.eval()
```

Comment:
```suggestion
```
```suggestion
```
If we call `train_step` and `eval_step` manually as opposed to `model(input)`, do we still need to call `model.train()` and `model.eval()`